types/jcof/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+	"compilerOptions": {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/jcof/tsconfig.json;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.